storage/bucket_test.go

```diff
@@ -157,6 +158,7 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 			},
 			PublicAccessPrevention: "enforced",
 		},
+		Updated:    time.Now().Format("2006-01-02T15:04:05Z07:00"),
```

Comment:
Does this actually work? Wouldn't time.Now() be later when the comparison is made?

Reply:
Given that this was BucketAttrs to RawBucket Test hence i have removed this.
Although a nice catch, but it was working somehow. Will check why.